Pipfile

```diff
@@ -11,23 +11,20 @@ name = "pypi"
 ### package deps
 aiohttp = "==3.7.4.post0"
 asn1crypto = "==1.4.0"
-# we pin this as the range specified in open-aea-ledger-cosmos is wide
-cosmpy = "==0.3.1"
```

Comment:
Why did we remove this and protobuf? Just for my info

Reply:
They were not being used anywhere in this service, so I thought we should not pin them anymore as they are also pinned in `open-aea`.

Moreover, they were causing dependency issues after bumping, so instead of upgrading them, I removed them. The dependency issues were caused because the new version of `open-aea`'s `Pipfile` now has:

[cosmpy = ">=0.4.1,<0.5.0"](https://github.com/valory-xyz/open-aea/blob/46191c12c30857d82a85d842ac2209c1535b4a43/Pipfile#L42) and [protobuf = ">=3.20,<=3.20.1"](https://github.com/valory-xyz/open-aea/blob/46191c12c30857d82a85d842ac2209c1535b4a43/Pipfile#L34).